packages/api/src/trpc.ts

```diff
@@ -7,11 +7,20 @@
  * The pieces you will need to use are documented accordingly near the end
  */
 import { initTRPC, TRPCError } from "@trpc/server";
+import axios from "axios";
 import superjson from "superjson";
 import { ZodError } from "zod";
 
 import type { Session } from "@forge/auth";
 import { auth, validateToken } from "@forge/auth";
+import { DISCORD_ADMIN_ROLE_ID } from "@forge/consts/knight-hacks";
+
+import { env } from "./env";
+
+interface User {
```

Comment:
Import this from the `auth` package. We want to avoid creating unnecessary types as much as possible.